pager/pager.go

```diff
@@ -32,12 +33,32 @@ func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		m.viewport.Width = msg.Width
 		textStyle := lipgloss.NewStyle().Width(m.viewport.Width)
 		var text strings.Builder
+
+		// Determine max width of a line
+		maxLineWidth := m.viewport.Width
+		if m.softWrap {
+			vpStyle := m.viewport.Style
+			maxLineWidth -= vpStyle.GetHorizontalBorderSize() + vpStyle.GetHorizontalMargins() + vpStyle.GetHorizontalPadding()
```

Comment:
Isn't there a better way to get the available width for the content? 
The previously used `m.viewport.Width` is the full width of the viewport.